src/main/java/seedu/trackermon/logic/commands/CommandResult.java

```diff
@@ -17,21 +17,33 @@
     /** The application should exit. */
     private final boolean exit;
```

Comment:
Maybe can make it sound more boolean-like: isExit?

Reply:
Noted, it was originally like that from AB3's code. I'll changed it.